chain/network/src/types.rs

```diff
@@ -838,12 +838,22 @@ pub enum NetworkRequests {
 }
 
 pub struct ValidateEdgeList {
```

Comment:
I'd also mention that this list is 'per peer update', right? (basically these are the edges that we've received from a given peer during a single update).

And consider moving source_peer_id to the top - to highlight this fact (as AFAIK this is not serialized anywhere)

Reply:
Ok, I moved `source_peer_id` to the top.
I also mentioned that those are edges received through  `NetworkRequests::Sync` or `NetworkRequests::IbfMessage`.